webapp/app/forms/steps/lotse/personal_data_steps.py

```diff
@@ -223,7 +223,8 @@ def get_religion_field():
 class StepPersonA(FormStep):
     name = 'person_a'
 
-    label = _l('form.lotse.step_person_a.label')
+    label = ngettext('form.lotse.step_person_a.label', 'form.lotse.step_person_a.label',
+                     num=1)
```

Comment:
I'm not sure why we have this? Is this just the default value in case we did not override get_label? Because I don't think we should need it?

Reply:
Yes, I think you are right. It is now set to None